server/notify.py

```diff
@@ -0,0 +1,87 @@
+from web3 import Web3

```

Comment:
Imported but not specified in requirements that I added . 
Have to create a new environment to install